assets/scss/style.scss

```diff
@@ -213,3 +215,106 @@ html, body {
 section.comments {
   margin-top: 2.5rem;
 }
+
+/* === All styles related to the author page, /author/author-name. === */
+
+$img-dimension: 20rem;
+$author-padding: 6rem;
+
+.main-header {
```

Comment:
I kept in the image dimension variable, used for the author image. Removed the padding though!